README.md

```diff
@@ -1,67 +1,89 @@
-- For Android-iOS, we need to connect to the iOS device via GATT to get the ID
-- We use [Firebase Phone Number Authentication](https://firebase.google.com/docs/auth/android/phone-auth) for verifying phone numbers and authentication
-- We use [Firebase Storage](https://firebase.google.com/docs/storage) for uploading user data in CSV format
-- We use [Firebase Functions](https://firebase.google.com/docs/functions) for getting the IDs to broadcast after registration
-- We use [Firebase Crashlytics](https://firebase.google.com/docs/crashlytics) for monitoring crashes from the app
+eRouška uses: 
+- Bluetooth Low Energy (BLE)
+- [Firebase Phone Number Authentication](https://firebase.google.com/docs/auth/android/phone-auth) for phone number verification and authentication
+- [Firebase Storage](https://firebase.google.com/docs/storage) for uploading the collected user data in CSV format
+- [Firebase Functions](https://firebase.google.com/docs/functions) for getting the _Device ID_'s to broadcast after registration
+- [Firebase Crashlytics](https://firebase.google.com/docs/crashlytics) for App crash monitoring
 
-More details:
-- [Technical documentation](https://github.com/covid19cz/erouska-android/wiki/Technical-documentation)
-- [iOS app code](https://github.com/covid19cz/erouska-ios)
-- [Firebase Functions code](https://github.com/covid19cz/erouska-firebase)
-- [Website code](https://github.com/covid19cz/erouska-homepage)
+- For Android <-> Android Bluetooth connection, the App doesn't need to connect to the other device as the _Device ID_ is broadcasted in the broadcast payload.
+- For Android <-> iOS Bluetooth connection, the App needs to connect to the iOS device via GATT to get the _Device ID_.
 
-## Who is behind the app?
+More details about eRouška:
+- [Technical documentation in Wiki](https://github.com/covid19cz/erouska-android/wiki/Technical-documentation)
+- [erouska-ios](https://github.com/covid19cz/erouska-ios): iOS source code
+- [erouska-firebase](https://github.com/covid19cz/erouska-firebase): Firebase Functions source code
+- [erouska-homepage](https://github.com/covid19cz/erouska-homepage): erouska.cz webpage source code
 
-We are initiative consisting from people from various Czech IT companies called [COVID19CZ](https://covid19cz.cz). We are all unpaid volunteers. We got backing from [Czech Ministry of Healthcare](https://www.mzcr.cz/).  
+## Building the App from the source code
 
-## Contributing
+Clone this repository and import the project into Android Studio.
 
-We are happy to accept pull requests. If you want to help more, discuss with us etc., please join [COVID19CZ](https://covid19cz.cz) Slack (channel #erouska), internally we use [Trello](https://trello.com/b/4xN2Eeqv/bug-wf) for bugs.
+Run:
+`./gradlew assembleDevDebug`
+
+## Contributing
+We are happy to accept pull requests! See [Git Workflow](#git-workflow).
 
-## Git flow
+We are on Slack! [covid19cz.slack.com](covid19cz.slack.com), channel _#erouska_.
```

Comment:
Mention that joining Slack is just for most active contributors, not for everyone.